_data/payment_config.yml

```diff
@@ -1,4 +1,4 @@
-orderUrl: https://secure.2checkout.com/order/checkout.php?PRODS=31007663&QTY=1&CART=1&CARD=1&SHORT_FORM=1&CURRENCY=EUR
+orderUrl: https://secure.2checkout.com/checkout/buy?merchant=999999999589&currency=EUR&tpl=default&prod=4HJME1JJDF&qty=1
```

Comment:
I'd rather extract `merchant`, `prod` into separate variables. It'd be just easier to work with them if they are not bundled into the link directly.

Reply:
@andreySadomovskiy as discussed, can we extract `merchant` and `prod` into separate variables and use within this `orderUrl`?

Reply:
We can't use variables in current Jekyll version. So I decided to leave the URL as is.